parachains/runtimes/testing/penpal/src/weights/block_weights.rs

```diff
@@ -0,0 +1,46 @@
+// This file is part of Substrate.
```

Comment:
Should part of 'Cumulus' probably?
Same in all files under 'weights' dir

Reply:
They all say that even for statemine. One would need to parameterise the generator if we really wanted to change that.